earth_enterprise/src/installer/install_server.sh

```diff
@@ -649,8 +649,13 @@ fix_postinstall_filepermissions()
   # chmod 644 $PUBLISHER_ROOT/stream_space/.config
   chmod 644 "$PUBLISHER_ROOT/.config"
   chmod 755 "$PUBLISHER_ROOT"
-  chown -R "$GEAPACHEUSER_NAME:$GRPNAME" "$PUBLISHER_ROOT/stream_space"
-  chown -R "$GEAPACHEUSER_NAME:$GRPNAME" "$PUBLISHER_ROOT/search_space"
+
+  SEARCH_OWNER=`find "$PUBLISHER_ROOT/search_space" -maxdepth 0 -printf "%g:%u"`
+  STREAM_OWNER=`find "$PUBLISHER_ROOT/stream_space" -maxdepth 0 -printf "%g:%u"`
+  if [ "$SEARCH_OWNER" != "$GEGROUP:$GEAPACHEUSER" -o "$STREAM_OWNER" != "$GEGROUP:$GEAPACHEUSER" ] ; then
+        printf "WARNING: The installer detected the publish root may have incorrect permissions! After installation you may need to run \n\
+sudo /opt/google/bin/geconfigurepublishroot --noprompt --chown --path=$PUBLISHER_ROOT"
```

Comment:
I think you need a `\n` on the end of this. This is the output I'm getting:

```
WARNING: The installer detected the publish root may have incorrect permissions! After installation you may need to run 
sudo /opt/google/bin/geconfigurepublishroot --noprompt --chown --path=/gevol/published_dbsStarting server
waiting for server to start.... done
```

There may be similar issues in the install_fusion script.

Reply:
I'm seeing this message print out on a fresh install where no /gevol exists before the install for some reason. I'm not sure why. The permissions are correct when it's created.